plugins/PassboltCe/AccountSettings/tests/TestCase/Controller/Themes/ThemesIndexControllerTest.php

```diff
@@ -32,20 +35,19 @@ class ThemesIndexControllerTest extends AppIntegrationTestCase
      */
     protected $AccountSettings;
 
-    public array $fixtures = [
-        'plugin.Passbolt/AccountSettings.AccountSettings',
-    ];
-
     public function setUp(): void
     {
         parent::setUp();
-        $this->AccountSettings = $this->fetchTable('AccountSettings');
+        $this->AccountSettings = $this->fetchTable('Passbolt/AccountSettings.AccountSettings');
     }
 
     public function testThemesIndexSuccess()
     {
+        $user = UserFactory::make()->user()->persist();
+        AccountSettingFactory::make()->theme('midgar')->withUser($user)->persist();
+
         // Authenticate as ada and list the themes
-        $this->logInAsUser();
+        $this->logInAs($user);
         $this->get('/account/settings/themes.json?api-version=v2');
         $this->assertResponseOk();
```

Comment:
I would suggest to add an assertion here, ensuring that the theme `midgar` is well returned in the response.